### PR TITLE
Updated Forest world

### PR DESCRIPTION
**Pull request by Hume2**

### CODE REVIEW

**Comment by maxteufel** [comment]
Could you summarize the changes, attach some before/after screenshots, and make the commit messages more consistent (e.g. `New level [...]` vs `data/levels/world2: new level: [...]`)?

**Comment by tobbi** [comment]
This PR needs to be part of the data repo.

**Comment by Hume2** [comment]
I'll close this PR anyway, because we got some respones in the contest. Don't delete the branch please, I need to extract these levels.
